test/client/components/CommunityPosts.test.js

```diff
@@ -60,14 +71,57 @@ describe('CommunityPosts', () => {
       location: {}
     }
 
-    const node = mount(<CommunityPosts {...props}/>, {
+    const node = mount(<CommunityPosts {...props} />, {
       context: {store, currentUser: {id: 6}},
-      childContextTypes: {currentUser: React.PropTypes.object}
+      childContextTypes: {currentUser: PropTypes.object}
     })
 
     expect(node.find('.post-editor').length).to.equal(0)
 
     expect(node.find('.request-to-join').text())
     .to.equal('You are not a member of this community. Request to Join')
   })
+
+  describe('ProfileCompletionModules', () => {
+    let node
+
+    beforeEach(() => {
+      const props = {
+        params: {id: community.slug},
+        location: {}
+      }
+      node = mount(
+        <CommunityPosts {...props} />, {
+          context: {store, currentUser, dispatch: store.dispatch},
+          childContextTypes: {
+            currentUser: PropTypes.object,
+            dispatch: PropTypes.func
+          }
+        }
+      )
+      mockActionResponse(updateUserSettings(), {})
+    })
+
+    it('displays bio module if no bio entered', () => {
+      expect(node.find(ProfileBioModule).length).to.equal(1)
+      expect(node.find(ProfileSkillsModule).length).to.equal(0)
+    })
+
+    it('saves bio', () => {
+      const bioTextArea = node.find(ProfileBioModule).first().find('textarea').first()
+      bioTextArea.simulate('change', {target: {value: 'test bio'}})
+      const saveButton = node.find(ProfileBioModule).first().find('button').first()
+      saveButton.simulate('click')
+      expect(store.getState().people.current.bio).to.equal('test bio')
+      // LEJ: How do I get the currentUser in context updating correctly
+      //      in this set up? Prefetch?
+      //
```

Comment:
i would skip the 'saves bio' test since you're covering it in the individual module, and just do the next three tests by modifying `currentUser` *before* you configure the store and mount the component.